cli/main.c

```diff
@@ -464,11 +462,10 @@ char * link_rate_conversion(uint8_t link_rate) {
          case 2:
               return "8.0G";
          case 3:
-              return "16G";
```

Comment:
One commit for one thing, obviously this change has nothing to do with whitespace cleanup, and either be squashed if this was introduced in this PR, or use another commit to fix it.

Reply:
Thanks, I will work to follow one-commit one change policy.